dcc-portal-ui/app/scripts/analysis/js/analysisLauncher.js

```diff
@@ -117,10 +117,6 @@
           }
           return true;
         });
-      } else if (type === 'phenotype') {
-        _this.filteredList = _.filter(SetService.getAll(), function (set) {
-          return set.type === 'donor';
-        });
```

Comment:
Would recommend we keep this explicit filter for 'phenotype', and throw a warning/error for the else since that's a branch that we're not supposed to be able to reach